architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -431,3 +618,198 @@ image:
   dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
 ----
 ====
+
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+[[image-stream-mappings-working]]
+=== Working with Image Streams
+
+The following sections describe how to use image streams and image stream tags. For more information on working with
+image streams, see xref:../../dev_guide/managing_images.adoc#dev-guide-managing-images[Managing Images].
+
+[[image-stream-mappings-working-getting]]
+==== Getting Information about Image Streams
+
+To get general information about the image stream and detailed information about all the tags it is pointing to,
+use the following command:
+
+----
+oc describe is/<image-name>
+----
+
+For example:
+
+----
+oc describe is/python
+
+Name:			python
+Namespace:		default
+Created:		About a minute ago
+Labels:			<none>
+Annotations:		openshift.io/image.dockerRepositoryCheck=2017-10-02T17:05:11Z
+Docker Pull Spec:	docker-registry.default.svc:5000/default/python
+Image Lookup:		local=false
+Unique Images:		1
+Tags:			1
+
+3.5
+  tagged from centos/python-35-centos7
+
+  * centos/python-35-centos7@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+      About a minute ago
+----
+
+To get all the information available about particular image stream tag:
+
+----
+oc describe istag/<image-stream>:<tag-name>
+----
+
+For example:
+
+----
+oc describe istag/python:latest
+
+Image Name:	sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+Docker Image:	centos/python-35-centos7@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+Name:		sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25
+Created:	2 minutes ago
+Image Size:	251.2 MB (first layer 2.898 MB, last binary layer 72.26 MB)
+Image Created:	2 weeks ago
+Author:		<none>
+Arch:		amd64
+Entrypoint:	container-entrypoint
+Command:	/bin/sh -c $STI_SCRIPTS_PATH/usage
+Working Dir:	/opt/app-root/src
+User:		1001
+Exposes Ports:	8080/tcp
+Docker Labels:	build-date=20170801
+----
+
+[NOTE]
+====
+More information is output than shown.
+====
+
+[[image-stream-mappings-working-adding]]
+==== Adding Additional Tags to an Image Stream
+
+To add a tag that points to one of the existing tags, you can use the `oc tag` command:
+
+----
+oc tag <image-name:tag> <image-name:tag>
+----
+
+For example:
+
+----
+oc tag python:3.5 python:latest
+
+Tag python:latest set to python@sha256:49c18358df82f4577386404991c51a9559f243e0b1bdc366df25.
+----
+
+Use the `oc describe` command to confirm the the image stream has two tags, one (`3.5`) pointing at the external Docker image and another tag (`latest`) pointing to a different tag in the same image stream.
```

Comment:
> and another tag (`latest`) pointing to a different tag in the same image stream.

and another tag (`latest`) pointing to the same image because it was created based on the first tag.